frontend/src/pages/server/login.vue

```diff
@@ -68,19 +68,27 @@ import { isEmpty } from 'lodash-es';
 import { UserDto } from '@jellyfin/sdk/lib/generated-client';
 import { getBrandingApi } from '@jellyfin/sdk/lib/utils/api/branding-api';
 import { getUserApi } from '@jellyfin/sdk/lib/utils/api/user-api';
-import { useRoute } from 'vue-router';
+import { getSystemApi } from '@jellyfin/sdk/lib/utils/api/system-api';
+import { useRoute, useRouter } from 'vue-router';
 import { useI18n } from 'vue-i18n';
 import { useRemote } from '@/composables';
 
 const { t } = useI18n();
 const route = useRoute();
+const router = useRouter();
 const remote = useRemote();
 const api = remote.sdk.oneTimeSetup(
   remote.auth.currentServer?.PublicAddress || ''
 );
 
 route.meta.title = t('login.login');
 
+try {
```

Comment:
Why is this needed here? Doesn't the interceptor catches this already?

Reply:
Nope, cause in this case it doesn't get a 401, just a NET_ETROR from Axios, hence the interceptor doing nothing and just throwing the error.